transformers/component_tests/median/configured_transformer_test.go

```diff
@@ -0,0 +1,206 @@
+			Expect(medianAddressErr).NotTo(HaveOccurred())
+
+			aAddress := "0xffb0382ca7cfdc4fc4d5cc8913af1393d7ee1ef1"
+			aAddressID, aAddressErr := shared.GetOrCreateAddress(aAddress, db)
+			Expect(aAddressErr).NotTo(HaveOccurred())
+
+			msgSenderAddress := "0x" + fakes.RandomString(40)
+			msgSenderAddressID, msgSenderAddressErr := shared.GetOrCreateAddress(msgSenderAddress, db)
+			Expect(msgSenderAddressErr).NotTo(HaveOccurred())
+
+			liftModel.ColumnValues[event.HeaderFK] = header.Id
+			liftModel.ColumnValues[event.LogFK] = liftLog.ID
+			liftModel.ColumnValues[event.AddressFK] = medianAddressID
+			liftModel.ColumnValues[constants.MsgSenderColumn] = msgSenderAddressID
+			liftModel.ColumnValues[constants.AColumn] = pq.Array([]string{aAddress})
+			insertErr := event.PersistModels([]event.InsertionModel{liftModel}, db)
+			Expect(insertErr).NotTo(HaveOccurred())
+
+			key := common.HexToHash("d96e792a1d8f411d1f279b1c9ea610f0bcad22e8356661708ae9e456b88582f0")
+			value := common.HexToHash("0000000000000000000000000000000000000000000000000000000000000001")
+			liftDiff := test_helpers.CreateDiffRecord(db, header, keccakAddress, key, value)
+
+			transformErr := transformer.Execute(liftDiff)
+			Expect(transformErr).NotTo(HaveOccurred())
+
+			var orclResult test_helpers.MappingResWithAddress
+			err := db.Get(&orclResult, `SELECT diff_id, header_id, address_id, a AS key, orcl AS value from maker.median_orcl`)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(orclResult.AddressID).To(Equal(strconv.FormatInt(medianAddressID, 10)))
+			test_helpers.AssertMapping(orclResult.MappingRes, liftDiff.ID, header.Id, strconv.FormatInt(aAddressID, 10), "1")
+		})
+	})
+
+	Describe("slot", func() {
+		It("reads a slot diff row and persists it", func() {
+			liftLog := test_data.CreateTestLog(header.Id, db)
+			liftModel := test_data.MedianLiftModelWithOneAccount()
+
+			medianAddressID, medianAddressErr := shared.GetOrCreateAddress(test_data.MedianEthAddress(), db)
+			Expect(medianAddressErr).NotTo(HaveOccurred())
+
+			aAddress := "0xffb0382ca7cfdc4fc4d5cc8913af1393d7ee1ef1"
+			_, aAddressErr := shared.GetOrCreateAddress(aAddress, db)
+			Expect(aAddressErr).NotTo(HaveOccurred())
+
+			msgSenderAddress := "0x" + fakes.RandomString(40)
+			msgSenderAddressID, msgSenderAddressErr := shared.GetOrCreateAddress(msgSenderAddress, db)
+			Expect(msgSenderAddressErr).NotTo(HaveOccurred())
+
+			liftModel.ColumnValues[event.HeaderFK] = header.Id
+			liftModel.ColumnValues[event.LogFK] = liftLog.ID
+			liftModel.ColumnValues[event.AddressFK] = medianAddressID
+			liftModel.ColumnValues[constants.MsgSenderColumn] = msgSenderAddressID
+			liftModel.ColumnValues[constants.AColumn] = pq.Array([]string{aAddress})
+			insertErr := event.PersistModels([]event.InsertionModel{liftModel}, db)
+			Expect(insertErr).NotTo(HaveOccurred())
+
+			key := common.HexToHash("6666d42004e50ff3badadb0de5fc3899f37d3399b72f940c3f476b66e30ff7ba")
```

Comment:
would be cool to use actual addresses/keys here so we could independently verify that the keys loader is working as expected